corehq/apps/groups/models.py

```diff
@@ -222,7 +235,7 @@ def get_reporting_groups(cls, domain):
             startkey=key,
             endkey=key + [{}],
             include_docs=True,
-            # stale=settings.COUCH_STALE_QUERY,
+            stale=settings.COUCH_STALE_QUERY,
```

Comment:
Did you look over all calls to `get_reporting_groups` to make sure stale queries are ok? If not, I'd leave this change out for now